src/main.rs

```diff
@@ -6,12 +6,15 @@ use futures::StreamExt;
 use hyper::{header, Body, Request, Response, Server, StatusCode};
 use reqwest::Client;
 use route_recognizer::Router;
-use std::{env, net::SocketAddr, sync::Arc};
+use std::{env, net::SocketAddr, sync::Arc, time::Duration};
+use tokio::{task, time::sleep};
 use tower::{Service, ServiceExt};
 use tracing as log;
 use tracing::Instrument;
 use triagebot::{db, github, handlers::Context, notification_listing, payload, EventName};
 
+const JOB_PROCESSING_CADENCE_IN_SECS: u64 = 60;
```

Comment:
At some point in the future I'd like to take `JOB_PROCESSING_CADENCE_IN_SECS` out into an env variable with a default (I tend to keep configuration separated from the code) but it's really unimportant right now.